index.html

```diff
@@ -71,3 +73,170 @@ <h1 class="home-title">Hey There.<br/>I’m Ye Min.</h1>
         </ul>
       </div>
     </section>
+    <section class="second-page">
+      <div class="recent-work-wrap">
+        <h1 class="recent-work-title">My Recent Works</h1>
+        <hr class="recent-work-line" />
+      </div>
+      <div class="recent-work-body">
+        <img class="recent-work-img" src="images/yoga.png" alt="Yoga Image" />
+        <div class="multi">
+          <div class="multi-post">
+            <h1 class="multi-post-title">Multi-Post Stories</h1>
+            <p class="multi-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. This has been the industry's standard dummy
+              text ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <ul class="multi-btns">
+            <li class="multi-btn">CSS</li>
+            <li class="multi-btn">HTML</li>
+            <li class="multi-btn">Bootstrap</li>
+            <li class="multi-btn">Ruby</li>
+          </ul>
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="six-wrap">
+        <div class="six-box-wrap">
+          <div class="professional">
+            <h3 class="six-title">Profesional Art Printing Data</h3>
+            <p class="six-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+          </div>
+          <ul class="six-btns">
+            <li class="six-btn">HTML</li>
+            <li class="six-btn">Bootstrap</li>
+            <li class="six-btn">Ruby</li>
+          </ul>
+          <button class="six-project-btn" type="button">See Project</button>
```

Comment:
- [ ] Kindly implement the button interactions as suggested by the previous reviewer as this will increase your portfolio user experience

![image](https://github.com/mryeminthu/Portfolio/assets/63473584/fb160dc9-4dc5-4ddf-a5c1-a0458765374b)